packages/expo-router/assets/native-tabs.module.css

```diff
@@ -0,0 +1,109 @@
+:root {
+  --expo-router-native-tabs-font-family: --apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
+}
+
+.nativeTabsContainer {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  overflow: hidden;
+}
+
+.tabContent {
+  flex: 1;
+  display: flex;
+  max-height: 100%;
+  max-width: 100%;
+}
+.tabContent[data-state="inactive"] {
+  display: none;
+}
+
+.navigationMenuRoot {
+  top: 24px;
+  left: 50%;
+  transform: translateX(-50%);
+  position: fixed;
+  z-index: 10;
+  display: flex;
+  background-color: var(--expo-router-native-tabs-background-color, #272727);
+  height: 40px;
+  border-radius: 25px;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 5px;
+  box-sizing: border-box;
+  margin: 0;
+  max-width: 90vw;
+  overflow-x: auto;
+}
+
+.navigationMenuTrigger {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  list-style-position: inside;
+  height: 100%;
+  background-color: transparent;
+  border: none;
+  margin: 0;
+  height: 100%;
+  border-radius: 20px;
+  padding: 0 20px;
+  cursor: pointer;
+  outline-color: var(--expo-router-native-tabs-tab-outline-color, #444444);
+  position: relative;
+}
+
+.navigationMenuTrigger[data-state="active"] {
+  background-color: var(--expo-router-native-tabs-active-background-color, #444444);
+}
+
+.tabText {
+  font-weight: var(--expo-router-native-tabs-font-weight, 500);
+  font-size: var(--expo-router-native-tabs-font-size, 15px);
+  font-family: var(--expo-router-native-tabs-font-family);
+  font-style: var(--expo-router-native-tabs-font-style, normal);
+  opacity: var(--expo-router-native-tabs-text-opacity, 1);
+  color: var(--expo-router-native-tabs-text-color, #8b8b8b);
+  white-space: nowrap;
+}
+
+.navigationMenuTrigger[data-state="active"] .tabText {
+  color: var(--expo-router-native-tabs-active-text-color, #ffffff);
+  font-size: var(--expo-router-native-tabs-active-font-size);
+}
+
+.navigationMenuTrigger:not([data-state="active"]) .tabText:hover {
+  opacity: var(--expo-router-native-tabs-text-hover-opacity, 0.6);
+}
+
+.tabBadge {
+  --expo-router-native-tabs-local-badge-size: var(--expo-router-native-tabs-badge-size, 20px);
```

Comment:
Just a nit, but think I'd get rid of "native" in the css var and just call it tabs. --expo-router-tabs-badge-size

In general I think we should shorten them.